### ISSUE TITLE
Silent Scans

### ISSUE THREAD

**Issue by aSimpleGuy** [tachiyomiorg/tachiyomi-extensions#7926]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### Source name

Silent Scans

### Source link

https://silentscans.blogspot.com

### Language

English

**Comment by h-hyuuga** [comment]
Is this actually a scanlator?

I either instantly get redirected to some unrelated site that gets blocked by my adblocker, or infinite page dialogs and sound alerts that seem to be trying to scam.

**Comment by alessandrojean** [comment]
I can access just fine with uBlock Origin, but they are just a blogspot page without a proper online reader.
